### DISCUSSION TITLE
Use Private Key JWT instead of Client Secret for `provider "auth0"`

### DISCUSSION THREAD

**Discussion by gojanpaolo** [auth0/terraform-provider-auth0#990] [Q&A]
Hello! Is it possible to use Private Key JWT authentication instead of Client Secret for configuring `provider "auth0"`? Thanks!

**Comment by duedares-rvj** [comment]
This feature is now supported. 
Ref: https://github.com/auth0/terraform-provider-auth0/releases/tag/v1.23.0

**Reply by gojanpaolo** [reply]
Awesome! Thank you! Will try this out soon
